### PR TITLE
Test if OperationMetadata tag exists

### LINKED ISSUE #2850: Impossible to import WMTS
Impossible to import this WMTS[1] because it doesn't contain the optional tag OperationMetadata [2].

If the OperationMetadata  tag doesn't exist the requestEncoding should be REST by default.

[1] http://www.ogc.stadt-zuerich.ch/mapproxy/wmts/1.0.0/WMTSCapabilities.xml
[2] https://github.com/camptocamp/ngeo/blob/master/src/modules/import/wmtsgetcap.js#L52

### PR DESCRIPTION
**Pull request by oterral**
Fix #2850

### CODE REVIEW

File: src/modules/import/wmtsgetcap.js [hunk 1]
```diff
@@ -49,8 +49,11 @@ exports = function(gettext, gettextCatalog, ngeoWmtsGetCapTemplateUrl) {
...
-        const getTileMetadata = getCap['OperationsMetadata']['GetTile']['DCP']['HTTP']['Get'][0];
-        const requestEncoding = getTileMetadata['Constraint'][0]['AllowedValues']['Value'][0];
+        let requestEncoding;
+        if ('OperationsMetadata' in getCap && 'GetTile' in getCap['OperationsMetadata']) {
+          const getTileMetadata = getCap['OperationsMetadata']['GetTile']['DCP']['HTTP']['Get'][0];
+          requestEncoding = getTileMetadata['Constraint'][0]['AllowedValues']['Value'][0];
+        }
         const layerOptions = {
           'layer': layer['Identifier'],
           'requestEnconding': requestEncoding
```
Comment (fredj) [inline src/modules/import/wmtsgetcap.js]: oups, the key name is `requestEnconding` but it should be `requestEncoding`

Comment (oterral) [reply src/modules/import/wmtsgetcap.js]: Nice catch.

**Comment by oterral** [comment]
Maybe the typo fix will break some WMTS. Let me try before merging this.

**Comment by oterral** [comment]
In fact this these 2 lines are useless because we do exactly the same in `ol.source.WMTS.optionsFromCapabilities`

**Comment by sbrunner** [comment]
The Travis job 5 failed because https://docs.angularjs.org/ is down ...

**Pull request merged**